### ISSUE TITLE
Make it easier to work with many programs

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/paper-land#193]
In the creator interface, more complicated projects are getting difficult to manage.

Take a look at this:
![image](https://github.com/phetsims/paper-land/assets/6396244/c97ecbb7-beb6-4bd6-920d-9a82d4aa0f10)

@terracoda suggested that the programs could be collapsible so that they can take up less space. We could also make the space bigger and let you zoom out much farther.

**Comment by jessegreenberg** [comment]
Also @brettfiedler do you have any thoughts about how to make this better?

**Comment by jessegreenberg** [comment]
The `more-zoom` branch has a change that lets us zoom way out. But it required a change to SceneryStack (phet-lib build) because mouse wheel input wasnt working well at these large zoom levels with scenery.AnimatedPanZoomListener. I want to test that on a Mac (hopefully with @terracoda) before pushing to dev/main.

**Comment by jessegreenberg** [comment]
I am abandoning the ideas of the more-zoom branch because it requires scenery level changes. We now have other improvements in Creator like collapsing programs, resizing the scenery display area, and so on.

One more idea that seems worth pursuing is the ability to logically cluster groups of programs together in the interface.

**Comment by jessegreenberg** [comment]
During design meeting today we decided that group functionality is likely not necessary. Lets see how the current features that were added (collapse, draggable columns).

During design meeting @brettfiedler also reported two additional issues that were fixed:
1) Dragging a program while zoomed in had incorrect transforms.
2) The expanded state of programs was not saved.

Closing this issue for now.
